api/src/config.rs

```diff
@@ -1194,11 +1194,15 @@ fn default_work_dir() -> String {
     ".".to_string()
 }
 
-pub fn default_batch_size() -> usize {
+fn default_merging_size() -> usize {
     128 * 1024
 }
 
-fn default_prefetch_batch_size() -> usize {
+fn default_batch_size() -> usize {
```

Comment:
Can we give a more precise name for these options, how about (not sure I'm understanding this correctly):

```
default_merging_size -> default_general_request_batch_size
default_prefetch_batch_size -> default_prefetch_request_batch_size
default_batch_size -> default_user_io_batch_size
```

Reply:
`merging_size` is different with `batch_size`. It just used the default value of `batch_size`.
Another two are renamed now, together with their associated variable names and function names.

Reply:
Seems `merging_size` is also a kind of `prefetch_batch_size`. I'll figure it out.

Reply:
There's multiple variable names for `merging_size`:
https://github.com/dragonflyoss/nydus/blob/e069235b7a6c52ebeb1c9343275f2258ac1c8187/api/src/config.rs#L871-L885
Now I'm keeping `PrefetchConfigV2::batch_size`, `FsPrefetchControl::merging_size`, `BlobPrefetchConfig::merging_size` as they are for Backward compatibility (since they implemented `Deserialize` trait), keeping `AsyncPrefetchConfig::merging_size` for consistency. The last two, `default_prefetch_request_batch_size()`, `FileCacheEntry::prefetch_request_batch_size()`, are renamed.